src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java

```diff
@@ -196,14 +195,18 @@ private SqlCommandParser() {
 		} else if (node instanceof SqlUseDatabase) {
 			cmd = SqlCommand.USE;
 			operands = new String[] { ((SqlUseDatabase) node).getDatabaseName().toString() };
-		} else if (node instanceof SqlDescribeTable || node instanceof SqlRichDescribeTable) {
+		} else if (node instanceof SqlRichDescribeTable) {
 			cmd = SqlCommand.DESCRIBE_TABLE;
-			if (node instanceof SqlDescribeTable) {
-				operands = new String[] { ((SqlDescribeTable) node).getTable().toString() };
-			} else {
-				// TODO describe extended
-				operands = new String[] { String.join(".", ((SqlRichDescribeTable) node).fullTableName()) };
+			// TODO support describe extended
+			String[] fullTableName = ((SqlRichDescribeTable) node).fullTableName();
+			StringBuilder escapedNameBuilder = new StringBuilder();
+			for (int i = 0; i < fullTableName.length; i++) {
+				if (i > 0) {
+					escapedNameBuilder.append('.');
+				}
+				escapedNameBuilder.append('`').append(fullTableName[i]).append('`');
```

Comment:
simplify these code as ` Stream.of(fullTableName).map(f -> "`" + f + "`").collect(Collectors.joining("."))`